### PR TITLE
Fix yardoc syntax in SettingsConcern

### PR DESCRIPTION
**Pull request by fonglh**
[ci skip]

### CODE REVIEW

File: app/models/concerns/settings_concern.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ def enabled_component_ids
 
   # Disable/Enable components
   #
-  # @params [Array<String>] the ids of all the enabled components
+  # @param ids [Array<String>] the ids of all the enabled components
```
Comment (allenwq) [inline app/models/concerns/settings_concern.rb]: Should be `@param [Array<String>] ids `

Now we put param name after type.

Comment (fonglh) [inline app/models/concerns/settings_concern.rb]: That will cause an `unknown parameter name` warning. See the last comment at http://stackoverflow.com/questions/11311973/warning-in-yard-with-param-tag

Comment (allenwq) [inline app/models/concerns/settings_concern.rb]: Huh ?  they say `@param [Type] name description`, so in this case should be `@param [Array<String>] ids description` ?

Comment (fonglh) [inline app/models/concerns/settings_concern.rb]: oops looks like they both work and yardoc checks that the name matches the one used in the function.

**Pull request merged**